packages/lit-dev-content/site/tutorials/content/carousel/08.md

```diff
@@ -0,0 +1,45 @@
+Now we're ready to add the animation. To do so, we'll import the `animate`
+directive from the `@lit-labs/motion` package.
+
+<aside class="warning">
+  Packages release under the <code>@lit-labs</code> umbrella aren't quite ready for
+  production. These projects may be experimental or incomplete, and they're
+  subject to breaking changes. See the <a href="https://lit.dev/docs/libraries/labs/">docs</a>
+  for more info.
+</aside>
+
+```ts
+import {animate} from '@lit-labs/motion';
+```
+
+<aside class="info">
+  Normally, you'd also need to add <code>@lit-labs/motion</code> to your
+  <code>package.json</code> but to speed things up, that's already been done.
+</aside>
+
+The `animate` directive will take care of animating the position. It's
+great for "tweening" between different rendered layouts. Add the `animate`
+directive to the "container" surrounding the slots:
+
+```ts
+return html`
+  <div class="fit"
+    ${animate()}
+```
+
+Note that even though we're setting the <code>left</code> property,
```

Comment:
Doesn't need addressing, why not markdown instead of `<code>`?

Reply:
Fixed.